app/views/global_error_wrapper.scala.html

```diff
@@ -90,7 +116,7 @@
 @footerLinks = {}
```

Comment:
Should this be @footer_links as per injection?

Reply:
As discussed no change required as Some(footer_links()) has been applied where necessary